### PR TITLE
fix(issues) Fix browser navigation on org issue list

### PR DESCRIPTION
**Pull request by markstory**
Fix browser navigation not working on org issue list. This also sets a default querystring value of `is:unresolved` to match project issues.

Refs APP-1004

### CODE REVIEW

File: src/sentry/static/sentry/app/views/organizationStream/overview.jsx [hunk 1]
```diff
@@ -104,6 +105,24 @@ const OrganizationStream = createReactClass({
     }
   },
 
+  componentWillReceiveProps(nextProps) {
```
Comment (lynnagara) [inline src/sentry/static/sentry/app/views/organizationStream/overview.jsx]: Is there anyway we can run this block of code in `componentDidMount` instead to keep it together with the rest of the refetching code. `componentWillReceiveProps` is getting deprecated.

**Comment by lynnagara** [comment]
Err, sorry I meant to say componentDidUpdate not cDM

**Comment by markstory** [comment]
@lynnagara I changed this around to not use any deprecated hook methods anymore.

File: src/sentry/static/sentry/app/views/organizationStream/overview.jsx [hunk 2]
```diff
@@ -113,37 +104,66 @@ const OrganizationStream = createReactClass({
...
+
+  getGroupStatsPeriod() {
+    let currentPeriod = this.props.location.query.groupStatsPeriod;
+    return STATS_PERIODS.has(currentPeriod) ? currentPeriod : DEFAULT_STATS_PERIOD;
+  },
+
+  getEnvironment() {
+    let {environments} = this.state.selection;
+    if (Array.isArray(environments)) {
+      return environments[0];
```
[the remark this answers is not in the document]
Comment (markstory) [reply src/sentry/static/sentry/app/views/organizationStream/overview.jsx]: Now that you point it out, this code won't work. We'll have to pass none until the organization group index endpoint has bulk update implemented.

**Review by lynnagara — APPROVED** [review]

**Pull request merged**